calian_gnss:
  base:
    ros__parameters:
      use_corrections: False
      corrections_source: "Ntrip"
      baud_rate: 230400
      unique_id: "35 40 82 9F 98"
    
calian_gnss:
  rover:
    ros__parameters:
      use_corrections: False
      baud_rate: 230400
      unique_id: "35 40 82 9F 98"

calian_gnss:
  gps_publisher:
    ros__parameters:
      use_corrections: False
      corrections_source: "Ntrip"
      baud_rate: 230400
      unique_id: "35 40 82 9F 98"

calian_gnss:
  gps:
    ros__parameters:
      use_corrections: False
      baud_rate: 230400
      unique_id: "35 40 82 9F 98"

calian_gnss:
  rtcm_handler:
    ros__parameters:
      key: "Wap5SA.O5VD4g:t2ltWLfva3gp5kituU5M4f9JL83BIwioTV1hGOjrSLU"
      channel: "Rtcm1"